charts/camunda-platform/charts/zeebe/templates/statefulset.yaml

```diff
@@ -80,6 +80,12 @@ spec:
           value: {{ .Values.ioThreadCount  | quote }}
         - name: ZEEBE_BROKER_GATEWAY_ENABLE
           value: "false"
+        {{- if .Values.global.multitenancy.enabled }}
+        - name: ZEEBE_GATEWAY_MULTITENANCY_ENABLED
```

Comment:
❌ I think you want to configure this here https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/charts/zeebe-gateway/templates/gateway-deployment.yaml